libs/sdk-common/src/invoice.rs

```diff
@@ -75,6 +75,12 @@ impl From<SystemTimeError> for InvoiceError {
     }
 }
 
+impl From<InvoiceError> for LnUrlError {
```

Comment:
Perhaps move this to to the lnurl module?

Reply:
Unfortunately Rust won't allow it:
> Only traits defined in the current crate can be implemented for arbitrary types

`InvoiceError` is defined here in `invoice.rs`.

Reply:
Not following. lnurl module still exists at the same crate as invoice.rs (sdk-common).

Reply:
Moved it in `lnurl/error.rs`, next to where `LnUrlError` is defined: f7a977ccd37912afbfdb9e710738daa4a5309f07